theia/api/models/imagery_request.py

```diff
@@ -8,17 +8,26 @@
 
 
 class ImageryRequest(models.Model):
-    adapter_name = models.CharField(max_length=64, null=False)
-    dataset_name = models.CharField(max_length=64, null=False)
+    adapter_choices = (
```

Comment:
I think `_choices` should be an array of tuples. 

```suggestion
      adapter_choices = [
        ('US Geological Survey', 'usgs')
    ]
```